app/controllers/concerns/client_sortable.rb

```diff
@@ -24,4 +28,22 @@ def cookie_filters
 
     cookies[filter_cookie_name] ? HashWithIndifferentAccess.new(JSON.parse(cookies[filter_cookie_name])) : {}
   end
+
+  def vita_partners_for_tagify
```

Comment:
this looks right to me, but it feels ripe for some unit testing. can you add some to `client_sortable_spec.rb`?